core/app/models/spree/reimbursement.rb

```diff
@@ -163,6 +163,23 @@ def return_all(created_by: nil)
       perform!(created_by: created_by)
     end
 
+    # The returned category is used as the category
+    # for Spree::Reimbursement::Credit.default_creditable_class.
+    #
+    # @return [Spree::StoreCreditCategory]
+    def store_credit_category
```

Comment:
Why is this an instance method? It doesn't look like it relies on the instance in any way. Is it so that it can be overridden by people looking to dynamically determine the store credit category.

Reply:
> Is it so that it can be overridden by people looking to dynamically determine the store credit category.

Yes, exactly. The original implementation was accepting a reimbursement to allow overrides based on it (although the implementation didn't actually use it) and I didn't want to break the promise if anyone did do that.

Reply:
👍 Right, that makes sense.